multiworld/core/image_env.py

```diff
@@ -116,12 +120,21 @@ def sample_goals(self, batch_size):
         goals['image_desired_goal'] = img_goals
         return goals
 
+    def compute_rewards(self, actions, obs):
```

Comment:
I guess this is going to mask all rewards from the environment right... plus in image-based environments it will subtract two images and take the norm even if its not eventually needed. Maybe instead if reward_type is not "image_distance" or "image_sparse" it should just not do any computation and keep the reward from the wrapped_env @vitchyr

Reply:
Yeah, I agree with Ashvin. I'm thinking we add "wrapped_env" as a reward type which delegates the computation to the wrapped class. We can make it the default too since it's the cheapest thing to do.

Reply:
Yupp thats what I ended up doing